src/webots/gui/WbView3D.cpp

```diff
@@ -1064,7 +1064,7 @@ void WbView3D::setWorld(WbSimulationWorld *w) {
 
   // Creates the fast mode overlay
   if (!mDisabled3dViewOverlay) {
-    mDisabled3dViewOverlay = new WbWrenFullScreenOverlay("Disabled 3D view", 128, true);
+    mDisabled3dViewOverlay = new WbWrenFullScreenOverlay("Hidden 3D view", 128, true);
```

Comment:
This is more consistent with the rest of the UI
```suggestion
    mDisabled3dViewOverlay = new WbWrenFullScreenOverlay("Hidden 3D View", 128, true);
```